### PR TITLE
Feature/multi theme support

### PR DESCRIPTION
**Pull request by mmcarvalho**
Add the ability to install multiple themes/plugins. 
Themes can be packaged with WP-Construct (genesis is, later will add a timber theme) or in a git repository. 
Git repositories can be installed as submodules or cloned.

### CODE REVIEW

**Review by pan-paryczek — APPROVED** [review]

File: .scripts/build.sh [hunk 1]
```diff
@@ -3,36 +3,174 @@
...
+  THEMENAME=$1
+  while true; do
+    read -p "Would you like to build '${THEMENAME}'? " yn
+    case $yn in
+      [Yy]* )
+        THEMEDIR="$ROOT/wp-content/themes/$THEMENAME"
+        echo  "${GREEN}Building $1 theme…${NC}"
+        cd "${THEMEDIR}"
+        composer install --no-interaction $OPT_COMPOSER
+        npm install
```
Comment (lzcalderaro) [inline .scripts/build.sh:93]: Isso é apenas uma sugestão: Poderia trocar para o yarn.

File: .scripts/build.sh [hunk 2]
```diff
@@ -3,36 +3,174 @@
...
+        npm run build --if-present
+        cd "${ROOT}"
+        continue 3;;
+      [Nn]* ) continue 3;;
+      * ) echo "Please answer yes or no.";;
```
Comment (lzcalderaro) [inline .scripts/build.sh]: Please answer y or n.

File: .scripts/build.sh [hunk 3]
```diff
@@ -3,36 +3,174 @@
...
+            THEMENAME=$(basename "$REPOSITORY")
+            THEMENAME="${THEMENAME%.*}" #removing extension
+            fetch_git $REPOSITORY $THEMENAME '/wp-content/themes/'
+            build_theme $THEMENAME
+          else
+            build_theme $opt
+          fi
+        done;;
+      [Nn]* ) break;;
+      * ) echo "Please answer yes or no.";;
```
Comment (lzcalderaro) [inline .scripts/build.sh]: Please answer y or n.

**Review by lzcalderaro — APPROVED** [review]

File: .scripts/build.sh (hunk 1 above)
Comment (mmcarvalho) [reply .scripts/build.sh:93]: That is a nice suggestion. Added to roadmap.

**Pull request merged**